job-service-db/src/main/resources/db/migration/V2.0.0/V2__job_service.sql

```diff
@@ -0,0 +1,54 @@
+--
```

Comment:
I think you had these files directly under `migration` before.  Why are the folders introduced?  We now have the version number in both the folder and the file?  Will the minor and patch versions always be 0?

Reply:
removed the folders. I wanted to match what was done on liquibase, but realized that it was done that way as each folder would contain a similarly named file `changelog.xml`